### PR TITLE
CDN + CLI

### PR DESCRIPTION
**Pull request by fsimonazzi**
Use CDN for static site. 
Use CLI for all instructions except assigning roles to users

### CODE REVIEW

File: src/ClientApp/readme.md [hunk 1]
```diff
@@ -1,64 +1,88 @@
...
+export STORAGE_ACCOUNT_NAME=<storage account name>
+
+# Create the storage account 
+az storage account create --name $STORAGE_ACCOUNT_NAME --resource-group $RESOURCEGROUP --location $LOCATION --kind StorageV2
+
+# Enable static web site support for the storage account
+az storage blob service-properties update --account-name $STORAGE_ACCOUNT_NAME --static-website --404-document 404.html --index-document index.html
+
+# Upload the web site
+az storage blob upload-batch --source ./ClientApp --destination \$web --account-name $STORAGE_ACCOUNT_NAME
```
Comment (ferantivero) [inline src/ClientApp/readme.md]: it doesn't seem to be aligned with https://github.com/mspnp/serverless-reference-implementation/pull/43

could you please consider rebasing on top of?

Comment (fsimonazzi) [reply src/ClientApp/readme.md]: I'd rather not commit to that specific order yet.

**Comment by fsimonazzi** [comment]
Addresses #45 and #46

File: src/ClientApp/readme.md (hunk 1 above)
Comment (fsimonazzi) [reply src/ClientApp/readme.md]: rebased to master after merging

**Comment by fsimonazzi** [comment]
Rebased to master

**Comment by ferantivero** [comment]
LGTM

**Pull request merged**